openedx/core/djangoapps/discussions/models.py

```diff
@@ -78,6 +78,7 @@ class Features(Enum):
             Features.BLACKOUT_DISCUSSION_DATES.value,
             Features.COURSE_COHORT_SUPPORT.value,
             Features.RESEARCH_DATA_EVENTS.value,
+            Features.PRIMARY_DISCUSSION_APP_EXPERIENCE.value,
```

Comment:
What is this for?

Reply:
This was missed previously. This feature is available for edx-legacy provider